README.md

```diff
@@ -1,5 +1,17 @@
-<!-- TODO: Hiya, Connector Author! Throughout this template, you'll find TODO comments to help guide you and ensure
-you've updated this README to be ready for users. If you have questions, shout at the docs team on #team-docs in Slack -->
+Hiya, Connector Author 👋
+
+Use these handy markdown template pages to generate documentation for your open source Hasura Data Connector.
+
+It's designed to help your users get going as quickly as possible with your connector and is comprised of this 
+README.md and a number of sub-documents in the `docs` directory. You should edit them as you see fit. 
+
+Look out for the inline comments, which aren't rendered, to help you fill in the blanks.
+
+To list your connector on the Hasura Connector Hub, you should make a PR to the 
+[Hasura NDC Hub](https://github.com/hasura/ndc-hub) repo.
+
+Happy documenting! 🚀
+`#deleteme`
```

Comment:
```suggestion
<!-- TODO:
Hiya, Connector Author 👋

Use these handy markdown template pages to generate documentation for your open source Hasura Data Connector.

It's designed to help your users get going as quickly as possible with your connector and is comprised of this 
README.md and a number of sub-documents in the `docs` directory. You should edit them as you see fit. 

Look out for the inline comments, which aren't rendered, to help you fill in the blanks.

To list your connector on the Hasura Connector Hub, you should make a PR to the 
[Hasura NDC Hub](https://github.com/hasura/ndc-hub) repo.

Happy documenting! 🚀
-->
```

I like the improvements to the instructions, but I'd recommend keeping this a comment for a couple of reasons:
1. We use that convention elsewhere, so we're teaching them here to look for them.
2. Anyone using a quickfix list can easily search for `TODO` to see what remains.
3. I can easily see this accidentally making its way through review with it included and ending up getting rendered on the hub.

Reply:
Then they can't see comments when they visit repo, so this valuable introduction as to what this repo is, is hidden. 

I doubt it would be deployed to hub - there is so much other stuff that they have to edit on this page also which shouldn't be deployed to hub.

Reply:
> so this valuable introduction as to what this repo is, is hidden

Okay, that's fair 👍